Misc/NEWS.d/next/Library/2025-12-10-09-16-46.gh-issue-142403.MCyTuL.rst

```diff
@@ -0,0 +1 @@
+Stop leaking the selector file descriptor, and release it when done.
```

Comment:
Are you sure that there is a leak of a file descriptor? The selectors used by `_sendfile_zerocopy()` don't use a file descriptor: https://github.com/python/cpython/issues/142403#issuecomment-3642962395